### ISSUE TITLE
 add share donation on donation success

### ISSUE THREAD

**Issue by righteousrenegade** [giv2giv/widget#1]
We need to create the ability to share via social networks (twitter, linkedin, facebook, g+, tumblr) when you have successfully made a donation to a charity. (replace the "Share your donation please" text with these share buttons)

Will need to find the appropriate apis and include them in this widget, rather than rely on any apis used in the web-frontend.

**Comment by righteousrenegade** [comment]
facebook share works for the most part, except that if we try to pass in the giv2giv charity page, facebook inserts "fb_ref=Default" in between the domain and #charities/[charity-id], ruining the link. We could skip this by just having the link point back to giv2giv.org, or the charity's own website, although pointing back to the charity website doesn't make as much sense, since we'd rather get people to come to giv2giv and donate that way.

thoughts? @michaelblinn, @biscuitehh

**Comment by michaelblinn** [comment]
So the link is er _not_ working OK, because of fb_ref=Default (For future, we may want to [look into using share rather than feed](http://stackoverflow.com/questions/28428103/facebook-feed-method-link-by-default-adding-fb-ref-default-at-the-end-how-to-rem). 

I think the root of the problem is here: http://stackoverflow.com/a/29674943 @biscuitehh is this just an inherent problem in using hasher?

**Comment by righteousrenegade** [comment]
Yeah, michael, the hash is definitely the problem in the url creation. we could probably try that solution mentioned in that stack overflow. or we could redo the URL creation. Idk if that's what hasher is doing or not. and changing the feed dialog to teh share dialog is fairly straightforward iirc, if we want to go that way.
